security/container_content.adoc

```diff
@@ -0,0 +1,53 @@
+[[security-content]]
+= Container Content
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:sourcedir: ../
+
+toc::[]
+
+[[security-content-whats-in-your-container]]
+== What's In Your Container?
+
+When it comes to security, what is inside your container matters. For some time
+now, applications and infrastructures have been composed from readily available
+components, many of which are open source packages such as the Linux operating
+system, JBoss Web Server, PostgreSQL and Node.js.
+
+Containerized versions of these packages are now also readily available, so that
+you do not have to build your own. But, as with any code you download from some
+external source, you need to know where they originally came from, who built
+them and whether there is any malicious code inside them. You need to know:
```

Comment:
The "You need to know" doens't match the list. I'd suggest "Ask yourself the following:" Or change the list to "If the contents of the container will compromise your infrastructure" "If there are any vulnerabilities in the application layer" and "If the runtime and operating system (OS) layers are up to date".